.circleci/config.yml

```diff
@@ -240,24 +267,27 @@ jobs:
       - image: cimg/node:16.15
```

Comment:
We can update this to be the same used in the solidity repo.
```suggestion
      - image: cimg/node:18.16
```

Reply:
I merged #716 so versions are up to date now.